.github/workflows/PR.yml

```diff
@@ -143,5 +126,39 @@ jobs:
       - name: Success!
         run: echo "All IntelliJ plugin verification checks passed!"
 
+  buildSample:
+    runs-on: macos-14
+    permissions:
+      contents: read
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v4
+        with:
+          distribution: 'zulu'
+          java-version-file: .github/workflows/.ci-java-version
+      - uses: gradle/actions/setup-gradle@v3
+        with:
+          build-root-directory: sample
```

Comment:
Don't need this since you're not running Gradle
```suggestion
```
https://github.com/gradle/actions/blob/c276584302d4e637967851a78a9e98c18a7d28da/setup-gradle/action.yml#L106-L109